src/mongo/client/connection_pool.cpp

```diff
@@ -110,7 +110,7 @@ void ConnectionPool::closeAllInUseConnections() {
 void ConnectionPool::_cleanUpStaleHosts_inlock(Date_t now) {
     if (now > _lastCleanUpTime + kCleanUpInterval) {
         for (HostLastUsedMap::iterator itr = _lastUsedHosts.begin(); itr != _lastUsedHosts.end();
-             itr++) {
+             ++itr) {
```

Comment:
When the result is not captured/used, the definition of the increment operator is inline, it seems unlikely that the compiler will generate different code for ++itr and itr++. Can you actually demonstrate a performance or codegen difference from this or any of the other identical changes?